static/modsecurity.sh

```diff
@@ -93,6 +93,6 @@ check_command sed -i 's|# SecDefaultAction "phase:2,nolog,auditlog,pass"|SecDefa
 
 if [ -f /etc/modsecurity/whitelist.conf ]
 then
-    echo "ModSecurity activated!"
+    print_text_in_color "$Cyan" "ModSecurity activated!"
```

Comment:
This is a sucessfull status, so Green.

Reply:
Totally agree. Thats also what I had in mind. Just wanted to make one change after another.